clippy.toml

```diff
@@ -1 +1,2 @@
-msrv="1.63.0"
\ No newline at end of file
+msrv="1.63.0"
+allow-print-in-tests=true
```

Comment:
🔧 having `println!()` in tests makes the outputs pretty messy so I don't think they should be allowed. Only exception would be for the example crates.